prem/DivAndSpan.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+<head>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>
+        DivAndSpan
+    </title>
+    <style>
+      
+        body {
+            width: 98.5vw;
+            height: 96.5vh;
+            border: 2px solid black;
+        }
+        .main-section {
+            position: absolute;
+            width: 50%;
+            height: 50%;
+            margin: auto;

```

Comment:
What is the need for having margin: auto?

Reply:
margin :auto  basically position the element to the center of the document.
But many may used it as margin: 0 auto which denotes that the margin-top and margin-bottom should be zero and auto is to divide the left and right margin automatically
But when i tried it, i dint get favorable outcome so i used "margin:auto" and  set all position to zero, by doing this i override the element and make it work like margin:0 auto